### PR TITLE
[Feature][e2e] Suggest e2e test adapt M1 chip

### PR DESCRIPTION
**Pull request by SbloodyS**
Related pr & issue:
https://github.com/apache/dolphinscheduler/pull/9077
https://github.com/apache/dolphinscheduler/issues/9068

### CODE REVIEW

File: development/en-us/e2e-test.md [hunk 1]
```diff
@@ -177,9 +177,15 @@ The rest are similar cases and can be understood by referring to the specific so
 https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases
 
 ##  III. Supplements
+When running E2E tests locally, First, you need to start the local service, you can refer to this page:
```
Comment (zhongjiajie) [inline development/en-us/e2e-test.md:180]: Could you add one blank line after title?

Comment (zhongjiajie) [reply development/en-us/e2e-test.md:180]: Same as zh docs.

**Review by zhongjiajie — APPROVED** [review]
Good adding, @Tianqi-Dotes Could you please may a double check?

File: development/en-us/e2e-test.md (hunk 1 above)
Comment (SbloodyS) [reply development/en-us/e2e-test.md:180]: I'' add it.

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**